server/testdriver/core/src/main/java/org/infinispan/server/test/core/TestSystemPropertyNames.java

```diff
@@ -33,4 +33,8 @@
     * The path to a local installation of the server
     */
    public static final String INFINISPAN_TEST_SERVER_DIR = "org.infinispan.test.server.dir";
+   /**
+    * The name of the JGroups stack. Defaults to `test-tcp`
+    */
+   public static final String INFINISPAN_TEST_SERVER_CLUSTER_STACK = "org.infinispan.test.server.cluster.stack";
```

Comment:
Why not use `infinispan.cluster.stack`?

Reply:
That change is going to be in https://github.com/infinispan/infinispan/pull/8276
For this PR, I just would like to fix the issue.